test/test_cases/send_erc1155_batch_from_schain_to_mainnet.py

```diff
@@ -35,8 +35,10 @@ def __init__(self, config):
         super().__init__('Send ERC1155 Batch from schain to mainnet', config)
 
     def _prepare(self):
-        amount = 2 * 10 ** 18
-        self.blockchain.recharge_user_wallet(self.config.mainnet_key, self.config.schain_name, amount)
+        sleep( 5 )
+        amountRecharge = 2 * 10 ** 18 # 2 * 10 ** 18
```

Comment:
It's the most useful comment I've ever seen. (-:

Reply:
Removed. Thanks.